WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/PrivacyRing/PrivacyBarViewModel.cs

```diff
@@ -61,13 +64,13 @@ private void CreateSegments(IExtendedList<PrivacyBarItemViewModel> list, IEnumer
 	{
 		list.Clear();
 
-		if (Width == 0d)
+		var coinCount = pockets.SelectMany(x => x.Coins).Count();
+
+		if (coinCount == 0d)
```

Comment:
Shouldn't the `Width == 0d` be taken into account too to keep the original behavior?

Reply:
Fixed! thanks!